openstack-helm/triliovault/values_overrides/victoria-ubuntu_focal.yaml

```diff
@@ -2,21 +2,21 @@
 images:
   tags:
     bootstrap: docker.io/openstackhelm/heat:victoria-ubuntu_focal
-    db_init: docker.io/openstackhelm/heat:victoria-ubuntu_focal
-    db_drop: docker.io/openstackhelm/heat:victoria-ubuntu_focal
+    db_init: docker.io/trilio/trilio-common:5.0.5-victoria-ubuntu_focal
+    db_drop: docker.io/trilio/trilio-common:5.0.5-victoria-ubuntu_focal
```

Comment:
is this a new image that will be used for db operations ?

Reply:
yes